src/_includes/layouts/base.njk

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>{{ title }} - {{ base[locale].gcds }}</title>
+    <title>{% block page_title %}{{ title }} - {{ base[locale].gcds }}{% endblock %}</title>
```

Comment:
Adding blocks to base template to be able to overwrite page title passed on page data. Same with 'content' block on line 34.